examples/typescript-mentors/mentors.ts

```diff
@@ -2,10 +2,11 @@ import { Ollama } from 'ollama-node';
 
 const mentorCount = 3;
 const ollama = new Ollama();
+type Mentor = { ns: string, char: string };
```

Comment:
Is this example a good candidate to remove? I think so if there isn't an api feature it highlights

Reply:
I think it’s a great example that not every app needs to be hard. I think the better ones to go are some of the modelfiles. They made sense at our start.